client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -249,7 +250,7 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 	function previewDesign( design: Design, styleVariation?: StyleVariation ) {
 		recordPreviewedDesign( { flow, intent, design, styleVariation } );
 
-		if ( ! design.is_virtual ) {
+		if ( ! design.is_virtual || ( design.is_virtual && ! design.preselected_style_variation ) ) {
```

Comment:
Nitpick: I think this can be simplified a little bit.
```suggestion
		if ( ! design.preselected_style_variation ) ) {
```